### PR TITLE
Update channels.csv

### PR DESCRIPTION
**Pull request by ProPontifex**
See title

### CODE REVIEW

**Comment by M7381** [comment]
lien

**Review by Alstruit — CHANGES_REQUESTED** [review]
blocklist.csv
 `638    "LibertyErotikaFHD.ru" is missing in the channels.csv`

channels.csv
` 26426  SMSDTVChannel18.us: "logo" with value "https://resources.finalsite.net/images/f_auto,q_auto,t_image_size_2/v15288[10](https://github.com/iptv-org/database/actions/runs/11770251804/job/32782417826#step:8:11)052/smsdorg/ivcuoxjonhnqoeogiwhf/eSchoolLogo.png" matches the inverted pattern: /,/`
 
` 26427  SMSDTVChannel2.us: "logo" with value "https://resources.finalsite.net/images/f_auto,q_auto,t_image_size_2/v1528810052/smsdorg/ivcuoxjonhnqoeogiwhf/eSchoolLogo.png" matches the inverted pattern: /,/`

**Comment by ProPontifex** [comment]
"LibertyErotikaFHD.ru" is missing in the channels.csv": it _is_ there but is actually based in the UK; that is why it is now listed as c/UK;c/RU

cannot find 26427 or 26427

File: data/channels.csv [hunk 1]
```diff
@@ -18152,259 +18152,259 @@ LiyuanTV.cn,Liyuan TV,,,,CN,,Huzhou,c/CN,zho,general,FALSE,,,,,https://i.imgur.c
...
-LogoWest.us,Logo West,,,,US,,,c/US,eng,,FALSE,,,,http://www.logotv.com/,https://upload.wikimedia.org/wikipedia/commons/thumb/2/2f/Logo_TV_2015.svg/512px-Logo_TV_2015.svg.png
-LohasChannel.tw,Lohas Channel,樂活頻道,,Lexi International Co. Ltd.,TW,,,c/TW,jpn;eng,xxx,TRUE,,,,http://www.goldentv.com.tw/index.php?m=Product&a=index&pc_code=1080113&pop=false,https://i.imgur.com/mXxR9RN.png
-LoHermidaTelevision.cl,Lo Hermida Television,Lo Hermida Televisión,,,CL,CL-RM,,s/CL-RM,spa,,FALSE,,,,https://lhtv.cl/,https://i2.paste.pics/2cfcf32766eb75a8fec96086bc741ec2.png
-LokaleOmroepZeewolde.nl,Lokale Omroep Zeewolde,LOZ,,,NL,,,c/NL,nld,general,FALSE,,,,https://www.lokaleomroepzeewolde.nl/,https://i.imgur.com/Dw4u9wT.png
+LOGOSChannel.cm,LOGOS Channel,,,,CM,,Yaounde,c/CM,fra,religious,FALSE,,,,https://www.facebook.com/LaParoleEcrite,https://i.ibb.co/KW8pBSj/Screenshot-20230807-140218.jpg
+LogosTV.us,Logos TV,,,,US,US-MD,Columbia,c/US;c/ES,spa,religious,FALSE,,,,https://logostv.es/,https://i.imgur.com/Kr3U3Ub.png
+LogosNorthridgeTV.us,Logos Northridge TV,,,,US,US-CA,Northridge,c/US,ara,religious,FALSE,,,,https://logoschannel.com/,https://i.imgur.com/h8DioXU.png
```
Comment (Alstruit) [inline data/channels.csv:18207]:
```suggestion
LogosTV.es,Logos TV,,,,ES,,,c/ES,spa,,FALSE,,,,,https://i.imgur.com/Kr3U3Ub.png
LogosTV.us,Logos TV,,,,US,US-CA,Northridge,c/US,ara,religious,FALSE,,,,https://logoschannel.com/,https://i.imgur.com/h8DioXU.png
```

Incorrect channel name. Be sure the channel's name matches on what is being broadcast.
Two instances of LogosTV exists. One is the Spain, another is from US.

File: data/channels.csv [hunk 2]
```diff
@@ -18152,259 +18152,259 @@ LiyuanTV.cn,Liyuan TV,,,,CN,,Huzhou,c/CN,zho,general,FALSE,,,,,https://i.imgur.c
...
-LokSabhaTV.in,Lok Sabha TV,,,Lok Sabha,IN,,,c/IN,eng,,FALSE,,,,https://loksabhatv.nic.in/,https://i.imgur.com/QavgfiK.png
-LokshahiNews.in,Lokshahi News,,,,IN,,,c/IN,mar,news,FALSE,,,,https://www.lokshahinews.com/,https://i.imgur.com/bpgdafu.png
-LOKTV.nl,LOK TV,,,,NL,,,c/NL,nld,,FALSE,,,,,https://i.imgur.com/gKOcxmL.png
-LOL.rs,LOL,,,Željko Mitrović,RS,,,c/RS,srp,lifestyle,FALSE,,,,http://www.pink.rs/lol,https://i.imgur.com/fPdlyJJ.png
-LOLAfrica.za,LOL Africa,,,AfricaXP,ZA,,,c/ZA,eng,,FALSE,,,,,https://i.imgur.com/mgUk4qB.png
-LollyKids.uk,Lolly Kids,,,Mimyuni Media Entertainment EOOD,UK,,,c/UK,eng,kids,FALSE,,,,https://www.mimyuni.com/,https://i.imgur.com/DOpgaBe.png
+LokSabhaTV.in,Lok Sabha TV,,,Lok Sabha,IN,,New Delhi,c/IN,eng,legislative,FALSE,,,,https://loksabhatv.nic.in/,https://i.imgur.com/QavgfiK.png
+LokshahiNews.in,Lokshahi News,,,Zora Traders LTD,IN,,Mumbai,c/IN,mar,news,FALSE,,,,https://www.lokshahinews.com/,https://i.imgur.com/bpgdafu.png
+LOKTV.in,LOK TV,,,,IN,,Patna,c/IN,tel,general,FALSE,,,,https://www.youtube.com/@loktvofficial,https://i.imgur.com/gKOcxmL.png
```
Comment (Alstruit) [inline data/channels.csv:18219]:
```suggestion
LOKTV.in,LOK TV,,,,IN,,Patna,c/IN,tel,general,FALSE,,,,https://www.youtube.com/@loktvofficial,https://i.imgur.com/gKOcxmL.png
LOKTV.nl,LOK TV,,,,NL,,,c/NL,nld,,FALSE,,,,,https://i.imgur.com/gKOcxmL.png
```
Incorrect Channel. Create a new instance instead.

**Review by Alstruit — CHANGES_REQUESTED** [review]

File: data/channels.csv (hunk 1 above)
Comment (ProPontifex) [reply data/channels.csv:18207]: I did: theyre both called Logos. Edit them as you wish.

Comment (ProPontifex) [reply data/channels.csv:18207]: Edited

File: data/channels.csv (hunk 2 above)
Comment (ProPontifex) [reply data/channels.csv:18219]: Done

**Review by Alstruit — DISMISSED** [review]

**Review by Dum4G — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**